hail_search/queries/base.py

```diff
@@ -461,9 +462,17 @@ def _add_entry_sample_families(self, ht, sample_data):
         ht = ht.transmute(family_entries=family_sample_index_data.map(lambda family_tuple: family_tuple[1].map(
             lambda sample_tuple: ht.family_entries[family_tuple[0]][sample_tuple[0]].annotate(**sample_tuple[1])
         )))
+        if not is_merged_ht:
+            ht = self._annotate_entry_filters(ht)
 
         return ht, sorted_family_sample_data
 
+    @staticmethod
+    def _annotate_entry_filters(ht):
```

Comment:
do we need to use the `family_sample_index_data` here to make sure filters are in the correct sample order?